pom.xml

```diff
@@ -101,6 +101,15 @@
         <scope>provided</scope>
       </dependency>
       <!-- End OpenMRS core -->
+
+      <dependency>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>legacyui-omod</artifactId>
+        <version>1.0</version>
```

Comment:
Create a property `<legacyuiVersion>1.0</legacyuiVersion>` to maintain the version with the others, see [here](https://github.com/openmrs/openmrs-module-bedmanagement/blob/28fb48c8a019e9257838f75e9aa61c3cb2d5cfc9/pom.xml#L38-L46).